typescript/eventbridge-lambda/lambda-handler.py

```diff
@@ -0,0 +1,2 @@
+def main(event, context):
+    print("I'm running!")
```

Comment:
It would be really nice if this did something Useful:tm:

Reply:
I have updated this to run a lambda that kicks off an sns topic and sends an email to the user who initiated the CDK stack